### ISSUE TITLE
iOS Live Photos

### ISSUE THREAD

**Issue by meichthys** [Piwigo/Piwigo#1677]
Support for iOS Live Photos would be a very welcome feature.

**Comment by relink2013** [comment]
Agreed. I was thrilled with the iOS app until I realized it doesn’t upload Live Photos.

**Comment by debug45** [comment]
I agree, it is a very important feature

**Comment by LioLynf** [comment]
Hi ! Just wanted to know if this kind of functionality is in developpement ?
Thanks a lot !

**Comment by Tonylolo** [comment]
Hi, have you any news about Live Photo player win Piwigo ?

thank you

**Comment by EddyLB** [comment]
@plegall could we add LivePhotos to version 15?
